### PR TITLE
Added CSS Grid CheatSheet & JS DOM CheatSheet

### PR DESCRIPTION
**Pull request by rahulraj22**
Made minor changes to other CSS flexbox cheatsheet and Js cheatsheet. 
Also included new file for CSS Grid cheatsheet and JS DOM cheatsheet.

### CODE REVIEW

File: CheatSheets/CSS Grid-CheatSheet.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+![JavaScript DOM Tutorial for Beginners](https://vishal-raj-1.notion.site/image/https%3A%2F%2Fs3-us-west-2.amazonaws.com%2Fsecure.notion-static.com%2F381724cd-61ee-4dda-9317-da3c96afc9c6%2Fcss-3.png?table=block&id=e3ef2276-f491-476f-bbc5-e17bc16074c5&spaceId=2119cbee-b8d9-4533-9b59-63ea95d76e4e&width=250&userId=&cache=v2)
```
Comment (Vishal-raj-1) [inline CheatSheets/CSS Grid-CheatSheet.md:1]: You add JavaScript Dom video in the CSS Grid Cheatsheet, change the video.

Comment (Vishal-raj-1) [inline CheatSheets/CSS Grid-CheatSheet.md:1]: CSS Grid Cheatsheet is already there in the home directory, Do remove home directory cheatsheet.

**Review by Vishal-raj-1 — CHANGES_REQUESTED** [review]
Do These changes

File: CheatSheets/CSS Grid-CheatSheet.md (hunk 1 above)
Comment (Vishal-raj-1) [reply CheatSheets/CSS Grid-CheatSheet.md:1]: Also, JavaScript DOM cheatsheet is also there,  you can pull the changes and update already written cheatsheet. Thanks

Comment (rahulraj22) [reply CheatSheets/CSS Grid-CheatSheet.md:1]: Actually, I added the correct link but by mistake I written wrong title.

File: CheatSheets/CSS Grid-CheatSheet.md [hunk 2]
```diff
@@ -1,154 +1,141 @@
...
-```css
-/* Aligns a grid item along the block (column) axis. */
-.item {
-  align-self: center;
-}
-```
-
-Learn More about CSS Grid from here:
-[![Learn CSS Grid in 35 minutes: The Ultimate Crash Course for Beginners]](https://www.youtube.com/embed/ULp7wPJ-rzQ "Learn CSS Grid in 35 minutes: The Ultimate Crash Course for Beginners")
+![CSS Grid Tutorial for Beginners](https://vishal-raj-1.notion.site/image/https%3A%2F%2Fs3-us-west-2.amazonaws.com%2Fsecure.notion-static.com%2F381724cd-61ee-4dda-9317-da3c96afc9c6%2Fcss-3.png?table=block&id=e3ef2276-f491-476f-bbc5-e17bc16074c5&spaceId=2119cbee-b8d9-4533-9b59-63ea95d76e4e&width=250&userId=&cache=v2)
```
Comment (Vishal-raj-1) [inline CheatSheets/CSS Grid-CheatSheet.md:1]: You have already added this at the end of the page.

File: CheatSheets/JavaScript DOM-Cheatsheet.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
+![JavaScript DOM Tutorial for Beginners](https://vishal-raj-1.notion.site/image/https%3A%2F%2Fs3-us-west-2.amazonaws.com%2Fsecure.notion-static.com%2F2ce8ae32-c03f-42db-9865-9a8abc50aa83%2Fjs.png?table=block&id=408271f3-22a5-4395-8ee8-755bc459674e&spaceId=2119cbee-b8d9-4533-9b59-63ea95d76e4e&width=250&userId=&cache=v2)
```
Comment (Vishal-raj-1) [inline CheatSheets/JavaScript DOM-Cheatsheet.md:1]: It is already added at the end of the file, it should be in the end only

**Review by Vishal-raj-1 — CHANGES_REQUESTED** [review]
Remove duplicate links

**Comment by Vishal-raj-1** [comment]
@rahulraj22, If you check CSS and JS DOM cheatsheet, you will find two video links in each file. Videos are present at top and bottom both places. You just have to remove from top place.

let me know if you need any help

**Pull request merged**